guacamole-common-js/src/main/webapp/modules/KeyEventInterpreter.js

```diff
@@ -283,7 +180,8 @@ Guacamole.KeyEventInterpreter = function KeyEventInterpreter(batchSeperation, st
         var name = String.fromCharCode(codepoint);
 
         // Create and return the definition
-        return new KeyDefinition({keysym: keysym, name: name, value: name, modifier: false});
+        return new Guacamole.KeyEventInterpreter.KeyDefinition({
+                keysym: keysym, name: name, value: name, modifier: false});
```

Comment:
`Guacamole.KeyEventInterpreter.KeyDefinition` does not appear to have a `modifier` property that would be copied from this template.

Reply:
Oops, indeed it doesn't. Removed.